CONTRIBUTING.md

```diff
@@ -22,8 +22,7 @@ Alternatively you may build from the command line using `msbuild.exe` or:
 
 ## Unity
 
-Unity Project requires several dependency DLL's. At first, run `copy_assets.bat` under `src\MessagePack.UnityClient`.
-Then open that directory in the Unity Editor.
+Then open `src\MessagePack.UnityClient` directory in the Unity Editor.
```

Comment:
"_Then_ open..." sounds like there's still a sentence that precedes it. But you removed it. Can we just change this to "Open..."?

Reply:
71c9b8194c1fad90ba0432a1b739a0ec9eb30436